src/isar/state_machine/states/monitor.py

```diff
@@ -86,3 +112,12 @@ def _log_status(self, mission_status: MissionStatus):
                 current_step=self.state_machine.status.current_mission_step,
             )
         self.iteration_counter += 1
+
+
+def mission_status_thread(
```

Comment:
Would it be better to have this function as part of the class?

Reply:
Implemented.